index.html

```diff
@@ -1,14 +1,259 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Mockup Home</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="\styles\style.css" rel="stylesheet" />
+</head>
+
+<body>
+  <section id="header">
+    <header>
+      <a href="/index.html">
+        <h1>HOME PAGE</h1>
+      </a>
+      <h6>The page for the home</h6>
+    </header>
+    <br>
+    <br>

```

Comment:
We want to avoid using the `<br>` tag for styling. A better way to go about this would be to handle it through CSS. Using things like margin or padding would handle this in a better way

Reply:
This comment also referrs to all of the other `<br>` tags through the project